lib/cc/analyzer/formatters/templates/html.erb

```diff
@@ -1,53 +1,697 @@
+        color: #9999a1;
+        margin-top: 10px;
+        margin-bottom: 15px;
+      }
+      .found-in a {
+        color: #35b0ff;
+      }
+      ::-webkit-details-marker {display: none;}
+      details summary {
+        display: block;
+      }
+      details summary::before {
+        content: "► ";
+
+      }
+      details[open] summary::before {
+        content: "▼ ";
+      }
+    </style>
   </head>
   <body>
-  <div class="container">
-    <h2>Code Climate Analysis</h2>
-
-    <h3><%= pluralize(@issue_count, "issue") %> found in <%= @issues_by_path.size %> files.</h3>
-
-    <% @issues_by_path.each do |path, file_issues| %>
-      <hr>
-      <h4><a name='<%= path %>'><%= path %></a> (<%= pluralize(file_issues.size, "issue") %>)</h4>
-      <hr>
-      <% file_issues.each do |issue| %>
-        <table class="table table-striped">
-          <tr>
-            <td>Description</td><td><%= issue["description"] %></td>
-          </tr>
-          <tr>
-            <td>Location</td><td><%= path %>:<b><%= issue["location"] %></td>
-          </tr>
-          <tr>
-            <td>Engine</td><td><%= issue["engine_name"] %></td>
-          </tr>
-          <tr>
-            <td>Category</td><td><%= issue["categories"].join(",") %></td>
-          </tr>
-          <tr>
-            <td>Remediation points</td><td><%= issue["remediation_points"] %></td>
-          </tr>
-        </table>
-        <% if issue["content"] %>
-          <br/>
-          <button data-toggle="collapse"
-                  data-target='#readup<%= issue['fingerprint'] %>'>
-                  Toggle Readup
-          </button>
-          <div id='readup<%= issue["fingerprint"] %>' class='collapse'>
-            <hr>
-            <%= issue["content"]["body"] %>
-            <hr>
-          </div>
-        <% end %>
-        <hr>
-      <% end %>
-    <% end %>
+    <header id="top">
+      <div class="container">
+        <a id="logo" href="https://codeclimate.com/">Code Climate</a>
+        <h1><%= h project_name %></h1>
+      </div>
+    </header>
+    <div class="container">
+      <nav>
+        <ul>
+          <li>
+            Issues
+          </li>
+        </ul>
+      </nav>
+      <div id="main-container">
+        <div class="issue-filters">
+          <label for="category-filter">Category</label>
+          <select id="category-filter">
+            <option value="">All Categories</option>
+            <%- categories.each do |category| -%>
+            <option value="<%= param category %>"><%= category %></option>
+            <%- end -%>
+          </select>
+          <label for="engine-filter">Engine</label>
+          <select id="engine-filter">
+            <option value="">All Engines</option>
+            <%- engines.each do |engine| -%>
+            <option value="<%= param engine %>"><%= engine %></option>
+            <%- end -%>
+          </select>
+        </div>
+        <ul id="smells">
+          <%- issues.each do |issue| -%>
```

Comment:
Would it be possible to get an empty state for when there are no issues to show?

There is, of course, the overall case of no issues having been found at all, but it's also possible to reach an "empty issues" state by selecting a combination of engine/category filters (e.g. style/duplication will yield no results for any repo that happens to include both style issues & duplication issues): it would be great if we could also show some copy for that instead of just a blank page.